templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -640,6 +605,8 @@ Mappings:
     Server:
       Name: registry.tanzu.vmware.com
   SampleAppProperties:
```

Comment:
SampleAppConfig &  SampleAppProperties can be totally removed and
./workload-aws.yaml file can be added as an src/resources directory

Reply:
these variables are used in a lot of places in the template...do i have to do this now for this to be merged? or can that be another story?

Reply:
@leclaist, please create a JIRA story to clean up all sample-app references properly. Also, I recommend using variable names instead of hardcoded values in the templates.

Once done, please resolve all comments here and then we can merge.